crates/subspace-node/src/commands/run.rs

```diff
@@ -338,8 +328,39 @@ pub async fn run(run_options: RunOptions) -> Result<(), Error> {
                             domain_start_options,
                         );
 
-                        if let Err(error) = start_domain.await {
-                            error!(%error, "Domain starter exited with an error");
+                        match start_domain.await {
+                            Ok(domain_code_executor) => {
+                                let span = info_span!("Consensus");
```

Comment:
We don't have to use the exact same executor,  creating an executor with the same type as `sc_domains::RuntimeExecutor` is also okay, see:
https://github.com/subspace/subspace/blob/60262348fe71b95f6cf1c25868f6100767a91ea3/crates/subspace-service/src/lib.rs#L485

Reply:
I would prefer we use same the executor across.